### PR TITLE
Python 3 compatibility

### PR DESCRIPTION
**Pull request by frensjan**
Hi!

@menno-tgho did some work in getting djangotoolbox compatible with python 3. Would be great if you could merge and push a new release to PyPi.

Cheers!
Frens

### CODE REVIEW

**Comment by r4fek** [comment]
Any chance to merge it soon?

File: djangotoolbox/db/base.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
-from django.utils.six.moves import cPickle as pickle
 import datetime
+import six
```
Comment (aburgel) [inline djangotoolbox/db/base.py]: `from django.utils import six`?

File: djangotoolbox/db/base.py [hunk 2]
```diff
@@ -11,8 +11,16 @@
...
+from django.utils.safestring import EscapeString, SafeString
+
+if six.PY3:
+    import pickle
+    EscapeUnicode = EscapeString
+    SafeUnicode = SafeString
+    StandardError = Exception
+else:
+    import cPickle as pickle
+    from django.utils.safestring import EscapeUnicode, SafeUnicode
```
Comment (aburgel) [inline djangotoolbox/db/base.py]: why not continue to import pickle from django.utils.six.moves?

**Comment by r4fek** [comment]
@aburgel what do you think about merging it to master?

File: djangotoolbox/fields.py [hunk 3]
```diff
@@ -81,11 +82,11 @@ def contribute_to_class(self, cls, name):
         super(AbstractIterableField, self).contribute_to_class(cls, name)
 
         # If items' field uses SubfieldBase we also need to.
-        item_metaclass = getattr(self.item_field, '__metaclass__', None)
-        if item_metaclass and issubclass(item_metaclass, models.SubfieldBase):
+        item_metaclass = type(self.item_field)
+        if issubclass(item_metaclass, models.SubfieldBase):
```
Comment (aburgel) [inline djangotoolbox/fields.py]: I don't think the `__metaclass__` is the same as an object's type.

**Comment by kavdev** [comment]
(This can be closed)

**Comment by aburgel** [comment]
Python 3 support was added in #63

**Pull request closed without merging**